### ISSUE TITLE
dnxcore50 in project.json

### ISSUE THREAD

**Issue by pantonis** [aspnet/Tooling#619]
I have a question. I just upgraded my system to .net core 1. I installed preview tooling 2 for VS2015
I created a class library (.NET Core) and my project.json files is

```
 {
  "version": "1.0.0-*",

  "dependencies": {
    "NETStandard.Library": "1.6.0"
  },

  "frameworks": {
    "netstandard1.6": {
      "imports": "dnxcore50"
    }
  }
}

```
why the dnxcore50 appears?

**Comment by barrytang** [comment]
TFS Bug#235750

**Comment by barrytang** [comment]
@Pilchie ?

**Comment by piotrpMSFT** [comment]
`dnxcore50` is imported because most NuGet packages folks want to use today were released before `Core 1.0` shipped [though it's nice to see how many package authors [have](https://www.nuget.org/packages/Newtonsoft.Json/9.0.1)/are already porting!]. Including the imports statement for `dnxcore50` is a point-in-time helper that makes older packages able to resolve in the context of your project. As more of the ecosystem moves on to netstandard* dependencies we will remove the imports statement from default templates.

**Comment by pantonis** [comment]
ok!
